test/signup.js

```diff
@@ -77,4 +79,37 @@ export async function verifyUserOverApi(username = 'automated@test.mail', passwo
     assert.equal(user.email, testUserEmail);
     assert.equal(user.first_name, testUserFirstName);
     assert.equal(user.last_name, testUserLastName);
+} */
+
+// This is a replacement of the above function `verifyUserOverApi` which requires the access token.
+// Note: access token can only requested after being loged in.
+export async function verifyUserByLogin(driver) {
+    await login(driver, testUserEmail, testUserPassword, verificationPageTitle);
+}
+
+export async function navigateToProfile(driver) {
+    await driver.get(`${url}/profile/profile`);
+
+    let title = await driver.getTitle();
+    assert.equal(title, 'Profile - Open Educational Badges');
```

Comment:
I think a wait until the title matches is more robust here. Also, why not use `const` for the title?

Reply:
Maybe we need also to add it everywhere.